scripts/upgrade

```diff
@@ -107,13 +117,14 @@ fi
 # Create the php-fpm pool config
 POOL_FPM
 
-# Setup SSOwat
-ynh_app_setting_set "$app" is_public "$is_public"
-if [ "$is_public" = "Yes" ];
-then
-  ynh_app_setting_set "$app" unprotected_uris "/"
+# Set ssowat config
+if [ "$is_public" = "Yes" ]; then
+	ynh_app_setting_set $app is_public 1 # Fixe is_public en booléen
+	is_public=1
+elif [ "$is_public" = "No" ]; then
+	ynh_app_setting_set $app is_public 0
+ 	is_public=0
 fi
 
-sudo systemctl reload php5-fpm
```

Comment:
I don't see why this wouldn't be needed any more... Some user may still use the previous version which wasn't using a dedicated php5-fpm.